node.js/package.json

```diff
@@ -7,7 +7,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@opencensus/exporter-zipkin": "0.0.7",
-    "@opencensus/nodejs": "0.0.7"
+    "@opencensus/exporter-zipkin": "0.0.19",
```

Comment:
Perhaps we should use caret version (`^`): will update you to the next minor version. `0.0.19` => `^0.0.19`.

Reply:
Changed